platform/brains2/src/lib/leds.cpp

```diff
@@ -147,11 +137,10 @@ static uint32_t show_pixels(const Pixel *const pixels, const int pixel_count) {
   return true;
 }
 
-void setBrightness(int brightness) {
-  _brightness = brightness;
-}
+void setBrightness(int brightness) { _brightness = brightness; }
 void show(const Pixel *const pixels, const int pixel_count) {
-  while(!show_pixels(pixels, pixel_count));
+  while (!show_pixels(pixels, pixel_count))
+    ;
```

Comment:
let's add an empty set of brackets here as recommended by the Google Style Guide.
https://google.github.io/styleguide/cppguide.html#Formatting_Looping_Branching